sdk/lib/crt/misc/crt_init.c

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
is this file exe specific?
Might make sense to include that in the name, so it's easier to differentiate.

Reply:
Well, it's a process-init, yes, and it does the same thing that crtdll / msvcrt do on process attach. But there is also thread-init, which I simply didn't bother to care about. And it should probably go into this file as well, if it was ever needed. Regarding the question, whether it's exe-specific: a DLL should never link statically to the CRT unless it's a CRT DLL like CRTDLL, MSVCRT, etc. The only real reason to link statically is to have tests that use our CRT even if they are run on Windows. There is really no other purpose. If you think there is a risk of people linking CRT statically into their new DLL, I could add a callback `int yes_i_know_i_should_not_link_crt_statically_but_i_know_what_i_am_doing(int x) { return x ^ MAGIC_STATIC_CRT_VALUE; }` if you think this is needed ;-)